### PR TITLE
Parsing priority

### PR DESCRIPTION
**Pull request by nparsons08**
⚡

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
 		"algoliasearch": "^3.26.0",
 		"axios": "^0.18.0",
 		"babel-plugin-inline-dotenv": "^1.1.2",
+		"bull": "^3.4.1",
```
Comment (kenhoff) [inline app/package.json]: can probably pull this - shouldn't be connecting to `bull` from the frontend app

File: api/src/controllers/rss.js [hunk 2]
```diff
@@ -112,17 +115,19 @@ exports.post = (req, res) => {
 			if (!feeds.feedUrls.length) {
 				return res
 					.status(404)
-					.send('We couldn\'t find any feeds for that RSS feed URL :(');
+					.send("We couldn't find any feeds for that RSS feed URL :(");
```
Comment (kenhoff) [inline api/src/controllers/rss.js]: by default, my `.prettierrc` always uses single ticks and then just escapes apostrophes. is this something that you'd like to enforce either way? if so, go ahead and pull this so we're not switching it back and forth every PR

File: app/src/components/AddPodcastModal.js [hunk 3]
```diff
@@ -25,14 +25,17 @@ class AddPodcastModal extends React.Component {
 		this.resetModal = this.resetModal.bind(this);
 		this.submitPodcastSelections = this.submitPodcastSelections.bind(this);
 	}
+
```
Comment (kenhoff) [inline app/src/components/AddPodcastModal.js]: go ahead and revert all changes to this file - isn't really part of this PR, can be changed in a separate cleanup PR

File: app/yarn.lock [hunk 4]
```diff
@@ -1642,6 +1642,18 @@ builtin-status-codes@^3.0.0:
   version "3.0.0"
   resolved "https://registry.yarnpkg.com/builtin-status-codes/-/builtin-status-codes-3.0.0.tgz#85982878e21b98e1c66425e03d0174788f569ee8"
 
+bull@^3.4.1:
```
Comment (kenhoff) [inline app/yarn.lock]: (this will probably need to be recommitted after removing `bull` from `app/package.json`)

File: app/package.json (hunk 1 above)
Comment (nparsons08) [reply app/package.json]: Thought this was pulled from package.json. Removing now.

File: api/src/controllers/rss.js (hunk 2 above)
Comment (nparsons08) [reply api/src/controllers/rss.js]: Yeah, I'm really not a fan of double quotes.

**Pull request merged**